test/hotspot/jtreg/runtime/CompressedOops/CompressedClassPointersEncodingScheme.java

```diff
@@ -71,6 +71,33 @@ private static void test(long forceAddress, boolean COH, long classSpaceSize, lo
         output.shouldContain("Narrow klass base: " + expectedEncodingBaseString + ", Narrow klass shift: " + expectedEncodingShift);
     }
 
+    private static void testFailure(String forceAddressString) throws IOException {
+        ProcessBuilder pb = ProcessTools.createLimitedTestJavaProcessBuilder(
+                "-Xshare:off", // to make CompressedClassSpaceBaseAddress work
+                "-XX:+UnlockDiagnosticVMOptions",
+                "-XX:CompressedClassSpaceBaseAddress=" + forceAddressString,
+                "-Xmx128m",
+                "-Xlog:metaspace*",
+                "-version");
+        OutputAnalyzer output = new OutputAnalyzer(pb.start());
+
+        output.reportDiagnosticSummary();
+
+        // We ignore cases where we were not able to map at the force address
+        if (!output.contains("Successfully forced class space address to " + forceAddressString)) {
```

Comment:
Ok.

Reply:
Oh the "forced" was because it was using CompressedClassSpaceBaseAddress which does force it to use this address (or else it exits).  So it is accurate.